src/conf_functions/recombination_functions.rs

```diff
@@ -1,12 +1,29 @@
 use crate::R;
 use rand::Rng;
 
+pub fn cut_point_recombine<const N: usize>(
+    parents_genes: (&[f64; N], &[f64; N]),
+    cut_point: usize,
+) -> ([f64; N], [f64; N]) {
+    let mut children_genes: ([f64; N], [f64; N]) = ([0.0; N], [0.0; N]);
+
+    for i in 0..cut_point {
+        children_genes.0[i] = parents_genes.0[i];
+        children_genes.1[i] = parents_genes.1[i];
+    }
+    for i in cut_point..N {
+        children_genes.0[i] = parents_genes.1[i];
+        children_genes.1[i] = parents_genes.0[i];
+    }
+    children_genes
+}
+
 pub fn random_cut_point_recombine<const N: usize>(
     parents_genes: (&[f64; N], &[f64; N]),
     rng: &mut R,
 ) -> ([f64; N], [f64; N]) {
     let cut_point = rng.gen_range(0..N);
-    let mut children_genes = ([0.0; N], [0.0; N]);
+    let mut children_genes: ([f64; N], [f64; N]) = ([0.0; N], [0.0; N]);
```

Comment:
Remove type and let the compiler infer it
```suggestion
    let mut children_genes: = ([0.0; N], [0.0; N]);
```

Reply:
sry for not catching that

Reply:
@stlenyk removed